addons/addons.json

```diff
@@ -157,6 +157,7 @@
   "asset-conflict-dialog",
   "remaining-replies",
   "forum-user-agent",
+  "mentions",
```

Comment:
Maybe this could be something like `highlight-mentions` since I have a mentions addon PR open as well?